<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalAccount</class>
 <widget class="QWidget" name="PersonalAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dandelion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">PersonalAccount
{
     background-color: rgb(202, 212, 255);
}</string>
  </property>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/arrow.png</normaloff>resources/arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>29</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Rating">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;rating&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VocabularyButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>251</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>vocabulary list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FlashcardsButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>flashCards</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
